tasks/WebRTCCommonTask.cpp

```diff
@@ -549,6 +549,11 @@ WebRTCCommonTask::Peer& WebRTCCommonTask::configureWebRTCBin(string const& peer_
     peer.last_signalling_message = base::Time::now();
 
     g_object_set(webrtcbin, "bundle-policy", m_signalling_config.bundle_policy, nullptr);
+    g_object_set(webrtcbin, "latency", 50, NULL);
```

Comment:
This can't be fixed. It has to come from a component property. The value here heavily depends on the network link.